src/common/entity/compute_state_display.ts

```diff
@@ -56,16 +56,19 @@ export const computeStateDisplayFromEntityAttributes = (
   }
 
   const domain = computeDomain(entityId);
-
+  const is_number_domain =
+    domain === "counter" || domain === "number" || domain === "input_number";
   // Entities with a `unit_of_measurement` or `state_class` are numeric values and should use `formatNumber`
   if (
     isNumericFromAttributes(
       attributes,
       domain === "sensor" ? sensorNumericDeviceClasses : []
-    )
+    ) ||
+    is_number_domain
   ) {
     // state is duration
     if (
+      !is_number_domain &&
```

Comment:
```suggestion
```
This check is not needed. The `device_class` check should be enough

Reply:
But then if a number entity has the class `duration` it will be formatted accordingly which looks a bit weird for the slider and also does not fit in a single line:

![image](https://github.com/user-attachments/assets/8e44f460-12cb-44f3-b82f-dad846303c8c)

Skipping the duration formatting keeps the state consistent with the blue bubble indicator.

Reply:
It is weird but that is state formatting issue. @piitaya will fix it separately